index.html

```diff
@@ -52,155 +52,177 @@ <h4 class="connect">LETS CONNECT</h4>
-                    <li><img src="images/works-dot.PNG" class="dot-separator" alt="Dot separator"></li>
-                    <li class="title-header-general">Back End Dev</li>
-                    <li><img src="images/works-dot.PNG" class="dot-separator" alt="Dot separator"></li>
-                    <li class="title-header-general">2015</li>
-                </ul>
-                <p class="project-description">
-                    A daily selection of privately personalized reads; no accounts or sign-ups required.
-                </p>
+            <div class="wider-screen-project-definition">
+                <div class="works-description">
+                    <h3 class="my-works-project-title">Tonic</h3>
+                    <ul class="works-headers">
+                        <li class="title-header">CANOPY</li>
+                        <li><img src="images/works-dot.PNG" class="dot-separator" alt="Dot separator"></li>
+                        <li class="title-header-general">Back End Dev</li>
+                        <li><img src="images/works-dot.PNG" class="dot-separator" alt="Dot separator"></li>
+                        <li class="title-header-general">2015</li>
+                    </ul>
+                    <p class="project-description">
+                        A daily selection of privately personalized reads; no accounts or sign-ups required.
+                    </p>
+                </div>
+                <section class="project-languages">
+                    <ul class="my-work-languages">
+                        <li class="coding-languages">html</li>
+                        <li class="coding-languages">css</li>
+                        <li class="coding-languages">javascript</li>
+                    </ul>       
+                </section>
+                <section class="project-more">
+                    <button class="works-see-more">See Project</button>
+                </section>  
             </div>
-            <section class="project-languages">
-                <ul class="my-work-languages">
-                    <li class="coding-languages">html</li>
-                    <li class="coding-languages">css</li>
-                    <li class="coding-languages">javascript</li>
-                </ul>       
-            </section>
-            <section class="project-more">
-                <button class="works-see-more">See Project</button>
-            </section>  
         </div>
         <div class="my-works">
-            <figure>
-                <img src="images/multi-post-stories-project.jpg" class="works-image" alt="Tonic App" width="100%"> 
+            <figure class="img-wide-screen flex-element-order">
+                <img src="images/multi-post-stories-project.jpg" class="works-image small-screen-view" alt="Facebook Project" width="100%"> 
+                <img src="images/Snapshoot-Portfolio-2.png" alt="Tonic project image" class="wide-screen-view">
```

Comment:
The project card photo disappears on the mobile screen, so please try to fix it.

![image](https://user-images.githubusercontent.com/34779928/193140942-fdba258f-01bc-4712-88c9-04dccdfa7617.png)

Reply:
Kindly, I suggest you review this comment from the previous reviewer, the portfolio images still show an error in the mobile version

Reply:
I have checked this a million times, it is working perfectly on my local computer. I will have a look at this again. thank you